core/include/drivers/atmel_saic.h

```diff
@@ -0,0 +1,47 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2021, Microchip
+ */
+
+#ifndef __DRIVERS_ATMEL_SAIC_H
+#define __DRIVERS_ATMEL_SAIC_H
+
+#include <tee_api_types.h>
+
+#define AT91_AIC_WPKEY	0x414943
+
+#define AT91_AIC_SSR		0x0
+#define	AT91_AIC_SSR_ITSEL_MASK	GENMASK_32(6, 0)
+
+#define AT91_AIC_SMR		0x4
+#define	AT91_AIC_SMR_SRC_SHIFT	5
+#define	AT91_AIC_SMR_LEVEL	(0 << AT91_AIC_SMR_SRC_SHIFT)
```

Comment:
It is generally more safe to use the macro `SHIFT_U32()` for this

Reply:
Ack